sfepy/solvers/eigen.py

```diff
@@ -445,6 +451,56 @@ def __call__(self, mtx_a, mtx_b=None, n_eigs=None, eigenvectors=None,
         return out
 
 
+class OctaveEigenvalueSolver(MatlabEigenvalueSolver):
+    """
+    Octave eigenvalue problem solver.
+    """
+    name = 'eig.octave'
+
+    def __init__(self, conf, comm=None, context=None, **kwargs):
+        from oct2py import octave as oe
+
+        EigenvalueSolver.__init__(self, conf, oe=oe, context=context,
+                                  **kwargs)
+
+    def solver_call(self, mtx_filename, eigs_filename, which):
+        import os
+        import scipy.io as sio
+
+        self.oe.addpath(os.path.dirname(__file__))
+        self.oe.matlab_eig(mtx_filename, eigs_filename)
+
+        evp = sio.loadmat(eigs_filename)
+        evals = evp['vals']
+
+        which = which.lower()
+        sort_funs = {
+            'lm': (nm.abs, -1),
+            'sm': (nm.abs, 1),
+            'la': (nm.real, -1),
+            'sa': (nm.real, 1),
+            'lr': (nm.real, -1),
+            'sr': (nm.real, 1),
+            'li': (nm.imag, -1),
+            'si': (nm.imag, 1),
+            'be': (nm.real, 1),
+        }
+
+        sfun, order = sort_funs[which.lower()]
```

Comment:
you don't need to use `.lower()` here. 

You are already using lower at line 476.

Same as line 492.

Reply:
Thanks, fixed.